ext/curl/curl_arginfo.h

```diff
@@ -233,7 +233,7 @@ static const zend_function_entry class_CurlShareHandle_methods[] = {
 static void register_curl_symbols(int module_number)
 {
 	REGISTER_LONG_CONSTANT("CURLOPT_AUTOREFERER", CURLOPT_AUTOREFERER, CONST_PERSISTENT);
-	REGISTER_LONG_CONSTANT("CURLOPT_BINARYTRANSFER", CURLOPT_BINARYTRANSFER, CONST_PERSISTENT);
+	REGISTER_LONG_CONSTANT("CURLOPT_BINARYTRANSFER", CURLOPT_BINARYTRANSFER, CONST_PERSISTENT | CONST_DEPRECATED);
```

Comment:
nit: needs to be done from the `ext/curl/curl.stub.php` perspective instead with the @deprecated annotation then using gen_stub.php on it.

Reply:
thanks, done https://github.com/php/php-src/pull/13114/commits/823c377f762b128e1e4ebc8f6f3dc1664e307b96